CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## X.X.X (YYYY-MM-DD)
 
+-   New option `kaoFiles` allowing to set a list of kao files that will used when initializing workspace's project parsing. This option is recommended for workspaces containing multiple sub-projects that are related such as a library and a test project.
```

Comment:
`+= "be"`
Edit after going further down the diff: a → the (it’s some kind of override, if I got it correctly. We _only_ parse those instead of looking for every `project.kao`.)
Edit 2: oh “a” is actually better now that I remember / understand.

```suggestion
-   New option `kaoFiles` allowing to set a list of kao files that will be used when initializing workspace's project parsing. This option is recommended for workspaces containing multiple related sub-projects, such as a library and a test project for this library.
```